examples/platforms/da15000/radio.c

```diff
@@ -41,25 +41,36 @@
 #include <hw_rf.h>
 #include <internal.h>
 #include <regmap.h>
-#include <hw_uart.h>
 
-#define DEFAULT_CHANNEL           (11)
+#define DEFAULT_CHANNEL                 (11)
+#define HARDCODED_NODE_ID               (1)
+
+#define PRIVILEGED_DATA                 __attribute__((section("privileged_data_zi")))
+
+#define IEEE802154_ACK_LENGTH           5
+#define IEEE802154_FRAME_TYPE_MASK      0x07
+#define IEEE802154_FRAME_TYPE_ACK       0x02
+#define IEEE802154_FRAME_PENDING        1 << 4
+#define IEEE802154_ACK_REQUEST          1 << 5
+#define IEEE802154_DSN_OFFSET           2
 
 static otInstance *ThreadInstance;
 
 static PhyState RadioState = kStateDisabled;
 static uint8_t Channel = DEFAULT_CHANNEL;
 static uint8_t TransmitPsdu[kMaxPHYPacketSize];
+static uint8_t ReceivePsdu[kMaxPHYPacketSize];
```

Comment:
style: statically-declared variable names should be prefixed with `s` (e.g. `sReceivePsdu` in this case).  Applies to definitions below as well.